components/range_cache_memory_engine/src/range_stats.rs

```diff
@@ -10,25 +10,40 @@ use std::{
     time::{Duration, Instant},
 };
 
+use collections::HashMap;
 use crossbeam::sync::ShardedLock;
 use kvproto::metapb::Region;
 use parking_lot::Mutex;
 use raftstore::coprocessor::RegionInfoProvider;
+use simple_moving_average::{SumTreeSMA, SMA};
```

Comment:
I saw the `SumTreeSMA` is only used to calculate a windowed average, it behavior is very like `Smoother` in `singleton_flow_controller.rs` or `SampleWindow` in `health_controller/trend.rs`. Maybe we can put one of them in tikv_util to avoid this unpopular external dependency.

Reply:
Good suggestion.